src/modules/test/recipes/SecurityBaselineTests.json

```diff
@@ -110,668 +110,10 @@
     "ComponentName": "SecurityBaseline",
```

Comment:
Why is this being reduced? Do they not work? If so, are we tracking it?

Reply:
This test recipe in its original form changes things (access rights to system and user files among them) and there is no VM that in default form can pass all audit checks. IN order to include this test recipe to the automated functional test for OSConfig, we need to clean it of things that are likely to fail and we'll add those bask as we advance through the implementation and we make things stable.